CHANGELOG.md

```diff
@@ -2,18 +2,21 @@
 
 This file documents all notable changes in `LogDNA CloudWatch Lambda Function`. The release numbering uses [semantic versioning](http://semver.org).
 
+## v2.0.2, v2.0.3 - Released on November 1, 2019
```

Comment:
I think I have released `v2.0.3` not on the same day

it's okay to have separate empty section for that as following:
```
## v2.0.3 - Released on November .., 2019
```